src/OSPSuite.Core/Domain/Services/QuantityValuesUpdater.cs

```diff
@@ -82,24 +82,24 @@ private void updateParameterValueFromStartValue(IModel model, PathAndValueEntity
          if (parameter == null)
             return;
 
+         //Formula is defined, we update in the parameter instance
          if (pathAndValueEntity.Formula != null)
          {
             parameter.Formula = _cloneManagerForModel.Clone(pathAndValueEntity.Formula);
+            
+            //ensures that the parameter is seen as using the formula
+            parameter.IsFixedValue = false;
             _keywordReplacerTask.ReplaceIn(parameter, model.Root);
          }
-         else
+
+         //If the value is defined, this will be used instead of the formula (even if set previously)
+         if (pathAndValueEntity.Value != null)
          {
-            var constantFormula = parameter.Formula as ConstantFormula;
-            var parameterValue = pathAndValueEntity.Value.GetValueOrDefault(double.NaN);
-            if (constantFormula == null)
-            {
-               parameter.Formula = _formulaFactory.ConstantFormula(parameterValue, parameter.Dimension);
```

Comment:
This is the line I don't get. Why would we replace the formula of a parameter with a constant. I think this does not make sense anymore

Reply:
I have no idea why this was done. 
Keeping the formula and setting the fixed value makes more sense